examples/Bremsstrahlung/include/simulation_defines/param/gridConfig.param

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2013-2017 Rene Widera, Richard Pausch, Benjamin Worpitz
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#pragma once
+
+namespace picongpu
+{
+
+    namespace SI
+    {
+        /** equals X
+         *  unit: meter */
+        BOOST_CONSTEXPR_OR_CONST float_64 CELL_WIDTH_SI = 8e-6 / 2048;
+        /** equals Y - the propagation direction
+         *  unit: meter */
+        BOOST_CONSTEXPR_OR_CONST float_64 CELL_HEIGHT_SI = CELL_WIDTH_SI;
+        /** equals Z
+         *  unit: meter */
+        BOOST_CONSTEXPR_OR_CONST float_64 CELL_DEPTH_SI = CELL_WIDTH_SI;
+        /** Duration of one timestep
+         *  unit: seconds */
+        BOOST_CONSTEXPR_OR_CONST float_64 DELTA_T_SI = CELL_WIDTH_SI / 3e8 / 1.4142135623730951 / 1.01;
```

Comment:
please add some documentation to the magic numbers `3e8 / 1.4142135623730951 / 1.01`